src/main/java/Reacher.java

```diff
@@ -0,0 +1,91 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Reacher {
+    public static void main(String[] args) {
+            Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> memory = new ArrayList<>();
+        System.out.println("Hello!\n" +
+                "I'm Reacher.\n" +
+                "Give me tasks.\n" +
+                "Functions are edit, list, delete and bye");
+        while (true) {
+            try {
+                String input = scanner.nextLine();
+                if (input.isEmpty()) {
+                    throw new ReacherException("pls type a task name.");
+                }
+                if (input.equalsIgnoreCase("bye")) { //check for end request
```

Comment:
While the if/else statements work properly to identify and separate logic blocks for different inputs, maybe you could also look into using switch/case statements or even enums which can look neater since there are so many different commands?